### PR TITLE
Add overview pages with diagrams

### PR DESCRIPTION
**Pull request by geoseong**
## Actions

- Add diagrams
  - client
  - server
- Add Overview doc includes diagram

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -5,73 +5,28 @@
...
+import { Badges, Contributors } from '../components';
+
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import React from 'react';
 import classnames from 'classnames';
-import styles from './styles.module.css';
+import styles from '../css/styles.module.css';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
```
Comment (hyochan) [inline src/pages/index.tsx]: Nope please do not change our main page it is designed our side.
Just applied in https://website.hackatalk.dev

**Review by hyochan — CHANGES_REQUESTED** [review]
I think the `overview` sections are great.
Please remove the main page updates by `rebasing` it.

Also please remove the contributors badges too.

**Comment by geoseong** [comment]
Okay I changed that you requested.

> I think the `overview` sections are great.
> Please remove the main page updates by `rebasing` it.
> 
> Also please remove the contributors badges too.

**Review by hyochan — APPROVED** [review]
Thanks for the great work!!

**Pull request merged**